python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -425,24 +425,22 @@ def run_optimize_model_gpu(repo_id,
     if repo_id in CHATGLM_IDS:
         model = AutoModel.from_pretrained(model_path, torch_dtype='auto', low_cpu_mem_usage=True,
                                           trust_remote_code=True, use_cache=True)
-        model = optimize_model(model, low_bit=low_bit)
         tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
-        model = model.to('xpu')
     elif repo_id in LLAMA_IDS:
         model = AutoModelForCausalLM.from_pretrained(model_path, trust_remote_code=True,
                                                      use_cache=True, low_cpu_mem_usage=True)
-        model = optimize_model(model, low_bit=low_bit)
         tokenizer = LlamaTokenizer.from_pretrained(model_path, trust_remote_code=True)
-        model = model.to('xpu')
     else:
         model = AutoModelForCausalLM.from_pretrained(model_path, torch_dtype='auto', low_cpu_mem_usage=True,
                                                      trust_remote_code=True, use_cache=True)
-        model = optimize_model(model, low_bit=low_bit)
         tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
-        model = model.to('xpu')
-        if isinstance(model, GPTJForCausalLM):
-            # For gpt-j model family, this optimization can provide a better performance.
-            model = ipex.optimize(model.eval(), inplace=True)
+
+    model = optimize_model(model, low_bit=low_bit)
+    model = model.eval()
+    if isinstance(model, GPTJForCausalLM):
+        # For gpt-j model family, this optimization can provide a better performance.
+        model = ipex.optimize(model, inplace=True)
+    model = model.to('xpu')
```

Comment:
need to put this line before the if block?

Reply:
sure